cmd/lncli/walletrpc_active.go

```diff
@@ -1193,6 +1195,195 @@ func listAddresses(ctx *cli.Context) error {
 	return nil
 }
 
+var signMessageWithAddrCommand = cli.Command{
+	Name:  "signmessage",
+	Usage: "Sign a message with the private key of the provided address.",
+	Description: `
+	Sign a message with the private key of the specified address, and
+	return the signature. Signing is solely done in the ECDSA compact
+	signature format. This is also done when signing with a P2TR address
+	meaning that the private key of the P2TR address (internal key) is used
+	to sign the provided message with the ECDSA format. Only addresses are
+	accepted which are owned by the internal lnd wallet.
+	`,
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "msg",
```

Comment:
See other comment about changing the order of the arguments/flags to match `bitcoin-cli` (`"address" "message"`).

Reply:
changed orders but kept short names for message = msg